troubadix/plugins/spelling.py

```diff
@@ -63,177 +60,193 @@ def run(self) -> Iterator[LinterResult]:
             codespell_ignore = f"{codespell_config_path}/codespell.ignore"
 
         # Run codespell as internal process
-        with redirect_stdout(io.StringIO()) as codespell:
-            codespell_main(
-                *(
-                    "--hard-encoding-detection",
-                    "--dictionary=-",
-                    f"--dictionary={codespell_additions}",
-                    f"--exclude-file={codespell_exclude}",
-                    f"--ignore-words={codespell_ignore}",
-                    "--disable-colors",
-                    f"{str(self.context.nasl_file)}",
-                )
+        batch_size = 10_000
+        for i in range(0, len(self.context.nasl_files), batch_size):
+            # files_parameter = [
+            #     str(nasl_file)
+            #     for nasl_file in self.context.nasl_files[i : i + batch_size]
+            # ]
+
```

Comment:
Always remove commented code if it is not documentation. It just confuses other devs. If you need some old code you can always look it up in git.
```suggestion
```

Reply:
True, forgot that